### PR TITLE
PRM-2599 made changes to integration tests for test data

### PR DESCRIPTION
**Pull request by RoheelArshad**

### CODE REVIEW

File: it/controllers/APIControllerISpec.scala [hunk 1]
```diff
@@ -53,7 +53,6 @@ class APIControllerISpec extends IntegrationSpecCommonBase with ETMPWiremock wit
         |       {
         |          "penaltyCategory": "LPP2",
         |          "penaltyStatus": "A",
-        |          "penaltyAmountAccruing": 123.45,
```
Comment (LewisMitchell) [inline it/controllers/APIControllerISpec.scala:56]: Accruing penalties have the penaltyAmountAccruing field

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: it/controllers/APIControllerISpec.scala (hunk 1 above)
Comment (RoheelArshad) [reply it/controllers/APIControllerISpec.scala:56]: Agree, but there was 2 of them...line 67 shows another one

Comment (LewisMitchell) [reply it/controllers/APIControllerISpec.scala:56]: Ah yes! Whoops!

**Comment by LewisMitchell** [comment]
Could you address this please?

> * Config key `auditing.traceRequests` can be removed from [conf/application.conf](https://github.com/hmrc/penalties/blob/PRM-2599/conf/application.conf#L80). It has been unused since play-auditing:3.0.0

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**